docker-compose.yml

```diff
@@ -33,6 +33,8 @@ services:
   worker:
     extends:
       service: app
+    depends_on:
+      - app
```

Comment:
This was needed to have the volumes mounted in the worker and scheduler containers that previously didn't work.